pages/clusters/queue_metrics.md

```diff
@@ -0,0 +1,24 @@
+# Queue metrics in clusters
+
+Queue metrics show the most important statistics to help you optimize your agent setup and monitor a cluster's performance. These statistics are updated on the page every 10 seconds.
+
+<%= image "cluster-queue-metrics.png", alt: "Screenshot of the queue metrics panel" %>
+
+## Agents panel
+
+_Agents Connected_ is the number of agents connected to the cluster queue. The circular chart represents the fraction of agents that are busy working on jobs compared to those that are idle and ready for a job. Hovering over the chart shows the _Agent Utilization_ panel, which displays the percentage values for each chart component.
+
+For agent utilization, agents are considered busy if they have a job ID assigned.
+
+<%= image "cluster-queue-metrics-agent-utilization.png", width: 530/2, height: 434/2, alt: "Screenshot of the agent utilization panel" %>
```

Comment:
I think the order of these should switch to put the image by the paragraph that introduces it.

```suggestion
<%= image "cluster-queue-metrics-agent-utilization.png", width: 530/2, height: 434/2, alt: "Screenshot of the agent utilization panel" %>

For agent utilization, agents are considered busy if they have a job ID assigned.
```